ArduPlane/Plane.h

```diff
@@ -506,6 +506,9 @@ class Plane : public AP_Vehicle {
 
         // how much correction have we added for terrain data
         float terrain_correction;
+        
+        // abort altitude target
+        int32_t abort_target_alt_cm;
```

Comment:
```suggestion
        int32_t abort_target_alt_agl_cm;
```

Reply:
This actually isn't AGL, it is relative. I can name it "abort_target_alt_rel_cm"

The most important thing is that the place where the abort climb termination condition is being checked uses the same altitude frame, and it does.